ada_2_fhir/fhir/2_fhir_fixtures.xsl

```diff
@@ -25,7 +25,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         <xd:desc>Creates xml document for a FHIR resource</xd:desc>
     </xd:doc>
     <xsl:template match="*" mode="doResourceInResultdoc">
-        <xsl:result-document href="./{./f:id/@value}.xml">
+        <xsl:result-document href="./{replace(./f:id/@value, '\.', '-')}.xml">
```

Comment:
Is dit de goede plek om dit te fixen? Ik dacht dat filename en logicalid overeen moeten komen (ivm FHIR IG) en dat is zo niet per definitie het geval. Moet dit niet eigenlijk in het fhir id zelf gedaan?

Reply:
Dit is een comment op een wijziging die ik later teruggedraaid heb, dus niet meer relevant (denk ik?)
In z'n algemeenheid ga ik nog even aan de slag met je opmerking om alles wat met Touchstone te maken heeft echt in aparte bestanden te bundelen